components/style/properties/gecko.mako.rs

```diff
@@ -738,6 +745,24 @@ fn static_assert() {
     }
     ${impl_simple_copy('box_sizing', 'mBoxSizing')}
 
+    % for value in GRID_LINES:
+    pub fn set_${value.name}(&mut self, v: longhands::${value.name}::computed_value::T) {
```

Comment:
- [Empty string, `0i32` and no span](https://dxr.mozilla.org/mozilla-central/rev/6dbc6e9f62a705d5f523cc750811bd01c8275ec6/layout/style/nsStyleStruct.h#1608-1621) should indicate `auto`, right?
- Clamping done :)